etl/steps/data/grapher/artificial_intelligence/2024-06-28/ai_corporate_investment.meta.yml

```diff
@@ -0,0 +1,37 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    processing_level: major
+    presentation:
+      topic_tags:
+        - Artificial Intelligence
+      grapher_config:
+        note: Data is expressed in constant 2021 US$. Inflation adjustment is based on the US Consumer Price Index (CPI).
```

Comment:
Minor: we suggest in the [Harmonizing FASTT guide](https://docs.google.com/document/d/1tcknTryJAxOR0yv8x38s3jkONq-TY3T4HNCEcZQ_c-s/edit?pli=1#heading=h.qd6mxo3fpwgz) to use `This data is expressed in constant 2021 US$`

Reply:
thank you!